src/content/reference/react/components.md

```diff
@@ -22,3 +22,23 @@ React exposes a few built-in components that you can use in your JSX.
 ## Your own components {/*your-own-components*/}
 
 You can also [define your own components](/learn/your-first-component) as JavaScript functions.
+
+--
+
+## Client and Server Components {/*client-and-server-components*/}
+
+<Wip>
+[React Server Components](/learn/start-a-new-react-project#bleeding-edge-react-frameworks) are only available in React’s canary and experimental channels. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+</Wip>
+
+There are two main types of components in React: Server Components and Client Components. Both types follow the same basic principles of React - they are reusable, composable, and stateful - but they are used in different scenarios and have different capabilities.
+
+### Server Components {/*server-components*/}
+
+Server Components are a new addition to React that allow developers to leverage the power of server-side rendering while maintaining the interactivity and richness of a client-side React application. They are rendered only on the server and have zero impact on the client-side bundle size, making them an excellent choice for parts of your application that are not interactive and don't need to be updated in response to user interaction.
+
+### Client Components {/*client-components*/}
+
+Client Components are the traditional React components that run on the client. They can access the full range of React features, including state, effects, and access to the DOM. They are essential for creating rich, interactive features where users expect immediate feedback.
```

Comment:
Not sure if this is temporary but we should be working towards server-first docs, not describing server components as a "new addition" and client components as "traditional"

Reply:
fixed.